packages/recommend-react/README.md

```diff
@@ -254,12 +254,9 @@ function App({ currentObjectID }) {
   return (
     <div className="auc-Recommendations">
```

Comment:
```suggestion
    <div className="auc-Recommend">
```

(there's other occurrences of this class)

Reply:
Good catch—fixed in 3c562450fc18f8f6628385764b4d6f1dddb0b0ad.